src/R/berekening_hulpvariabelen.R

```diff
@@ -239,4 +239,54 @@ bereken_vvi <- function(point,
     output_type = output_type)
 }
 
+add_visibility_to_frame <- function(punten_sf,
+                                    gebied,
+                                    name,
+                                    resolution,
+                                    spacing,
+                                    dist = 300,
+                                    obs_height = 1.7,
+                                    dsm_offset = 0) {
+
+  filename_dsm <- paste0("dhmvii_dsm_1m_", name, ".tif")
+  file_dsm <- file.path(mbag_dir, "data", "dem", filename_dsm)
+
+  if (file.exists(file_dsm)) {
+    dsm <- terra::rast(file_dsm)
+    crs(dsm) <- "epsg:31370"
+  } else {
+    dsm <- terra::rast("S:/Vlaanderen/Hoogte/DHMVII/DHMVIIDSMRAS1m.tif") %>%
+      terra::crop(gebied, filename = file_dsm)
+    crs(dsm) <- "epsg:31370"
```

Comment:
Eventueel alternatief is hier dat we de crop niet doen en op voorhand de dsm en dtm van Vlaanderen naar resolutie 5 x 5 m brengen. Dat is wellicht ongeveer 1 Gb voor heel Vlaanderen.